manager/manager.go

```diff
@@ -628,6 +628,11 @@ func (self *manager) AllDockerContainers(query *info.ContainerInfoRequest) (map[
 	for name, cont := range containers {
 		inf, err := self.containerDataToContainerInfo(cont, query)
 		if err != nil {
+			// Ignore the error because of race condition and return best-effort result.
+			if strings.Contains(err.Error(), "unable to find data in memory cache") {
```

Comment:
Define this error as a new type of error, and then use type assertion here, rather than string parsing.

Reply:
done